railties/lib/rails/generators/rails/app/templates/config/application.rb.tt

```diff
@@ -27,8 +27,9 @@ module <%= app_const_base %>
     config.load_defaults <%= Rails::VERSION::STRING.to_f %>
 
     # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
+    # Application configuration can go into files in config/initializers
+    # -- all .rb files in that directory are automatically loaded after loading
```

Comment:
Now that we're improving this, I wonder if we should mention that the files are loaded _alphabetically_?

> ... are automatically loaded _alphabetically_ after loading the framework ...

Reply:
Thanks! 👍 
I think the concern of here is "It can go into files in `config/initialisers`", not "They are loaded in alphabetically order".
It may be good, however, I think this should be kept as simple as possible.
Which do you think is better? I will add it if it is necessary.